packages/react/portal/src/Portal.tsx

```diff
@@ -58,6 +58,8 @@ const Portal = React.forwardRef((props, forwardedRef) => {
             ? {
                 position: 'absolute',
                 top: 0,
+                right: 0,
+                bottom: 0,
```

Comment:
This looks like it will put a layer on top of everything... which would break `disableOutsidePointerEvents={false}` on things right? because this element will prevent click through?

Reply:
Ooooh, I didn't think of this